### ISSUE TITLE
Rendering a PDF emits warning with 1.5.8

### ISSUE THREAD

**Issue by robertpanzer** [asciidoctor/asciidoctorj#707]
When rendering a PDF with the latest 1.5.8 I get a warning:
```
Okt 29, 2018 6:02:24 PM <script> (root)
WARNUNG: gem 'concurrent-ruby' is not installed. This gem is recommended when registering custom converters.
```
Apparently I forgot to add this gem.

@mojavelinux Is this critical? Should I prepare a 1.5.8.1 release?

**Comment by mojavelinux** [comment]
I forgot to give you a heads up about this. Asciidoctor 1.5.8 no longer uses thread_safe. That gem has been deprecated. It now uses concurrent-ruby. It provides the same functionality, but does not offer a compatible API. For details, see https://github.com/asciidoctor/asciidoctor/pull/2822.

**Comment by robertpanzer** [comment]
So do you think we need a new release?

I guess we will have to keep thread_safe in the asciidoctorj.jar because asciidoctorj-pdf relies on it being available.

**Comment by mojavelinux** [comment]
> So do you think we need a new release?

Yes, I think it's worth it.

> I guess we will have to keep thread_safe in the asciidoctorj.jar because asciidoctorj-pdf relies on it being available.

Correct. Both are currently needed.

**Comment by robertpanzer** [comment]
Fixed with 1.5.8.1
